proto/parca/profilestore/v1alpha1/profilestore.proto

```diff
@@ -60,6 +60,9 @@ message LabelSet {
 message RawSample {
   // raw_profile is the set of bytes of the pprof profile
   bytes raw_profile = 1;
+  // base_addresses contains base addresses for each pprof mapping
+  // found in the raw profile in pprof format.
+  repeated uint64 base_addresses = 2;
```

Comment:
Let's try to omit this from this level and try to use the data from the mapping level. It would be cleaner.
We can pass down the mapping to liners for example. That way we can omit passing `isRaw` to the `PCToLine`.

What do you think?

Reply:
> try to use the data from the mapping level

From what I understand the agent sends raw profiles in pprof format, so we can't add a base address field to pprof's `Mapping` message https://github.com/google/pprof/blob/main/proto/profile.proto#L147 since we don't control it. The server defines its own `Mapping` message to store received profiles in metastore. The conversion from pprof to an internal mapping format happens in `NormalizeMappings()` during pprof ingestion.

I thought that `RawSample` is the best candidate to pass base addresses without breaking backward compatibility 🤔.

Reply:
@kakkoyun I've moved `isRaw` field from `Location` to `Mapping` protobuf message to omit passing `isRaw` to the `PCToLine`.